packages/core/useMousePressed/index.ts

```diff
@@ -14,6 +14,12 @@ export interface MousePressedOptions extends ConfigurableWindow {
    */
   touch?: boolean
 
+  /**
+   * Listen to right mouse 'mousedown' event
+   * @default true
+   */
+  right?: boolean
```

Comment:
I would prefer a more generic options like: `butttons: ['left', 'right', 'middle']` to do the filtering.